data-structures-stacks/get-top.js

```diff
@@ -0,0 +1,6 @@
+export default function getTop(stack) {
+  if (!stack) {
```

Comment:
This check isn't really necessary since you should never be passed an `undefined` stack.